### ISSUE TITLE
Failing test is missing all information in Allure Report

### ISSUE THREAD

**Issue by cee-dee** [KasperskyLab/Kaspresso#548]
**Observed behavior**

In a succeding test, I get a lot of information on the test execution:
<img width="413" alt="succeding" src="https://github.com/KasperskyLab/Kaspresso/assets/12291958/67a4ee04-6cc9-4a0e-9feb-c9ac959b329f">

The above "Test Body" includes a description of the steps in the test as well a Logcat dump and a video recording of the test run.

In a failing test, I only get a dump of the View Hierarchy, nothing more:
<img width="413" alt="succeding" src="https://github.com/KasperskyLab/Kaspresso/assets/12291958/8608fd88-131e-4616-88ca-06601574f1fe">

I checked the `allure-results` folder downloaded from the device and it contains the video.

**Expected behavior**

Allure Reports show me all steps in the test execution (so far), even if a test has failed.

Is this an issue that can be solved via configuration or does it need an update of Kaspresso?

**Comment by Nikitae57** [comment]
Hi! Could you describe your configuration, please? 
1. Which kaspresso builder do you use
2. Test runner
3. Kaspresso version
4. System image
5. Is it an emulator or a real device

And it would be great to have a reproducable sample, if possible

Thank you!

**Comment by cee-dee** [comment]
Sure:
1. `kaspressoBuilder = Kaspresso.Builder.withForcedAllureSupport()`
2. `com.kaspersky.kaspresso.runner.KaspressoRunner`
3. version 1.5.2
4. `system-images;android-33;google_apis;arm64-v8a`
5. it's an emulator

While trying to produce a sample for reproduction, I struggled. I checked some older test runs where some other flaky tests failed -- they gave me actually the expected full entry in the Allure Report.

Would it help you, if I provide you with the generated Allure Results?
